CacheMultimap.adoc

```diff
@@ -0,0 +1,79 @@
+= Native cache Mutimap
+
+The goal is to provide Infinispan Native CacheMultimap support. The proposal here is to start with the smallest interface implementation, in order to provide a proper API to the first multimap client, vert-x project.
+However, this API will evolve, and this first implementation must work on Embedded and Server mode.
+This first Multimap won't support duplicate values on the same key. 
+
+[source, java]
+.CacheMultimap.java
+----
+public interface CacheMultimap<K, V> {
+   boolean put(K key, V value); // <1> <2>
+
+   Collection<V> get(K key); // <3>
+
+   boolean remove(K key); //<4>
```

Comment:
@rvansa -1 for option 3. A boolean return value tells the user whether a `key->value` mapping was added or not (because it already existed). I'm not sure how useful that would be, but the prior collection size seems pretty useless.

@karesti I wouldn't expose the full Cache API. Users that need more complex processing of the values, e.g. full-blown `compute()`, should use `Cache<K, Collection<V>>` directly.